architecture/additional_concepts/admission_controllers.adoc

```diff
@@ -25,6 +25,19 @@ system configured defaults. In addition, admission control plug-ins may modify
 related resources as part of request processing to do things such as
 incrementing quota usage.
 
+
+ifdef::openshift-aro[]
+[WARNING]
+====
+The {product-title} master has a list of plug-ins that are enabled by
+default for each type of resource (Kubernetes and {product-title}). These are
+required for the proper functioning of the master. Future versions of the
```

Comment:
Given that the docs are versioned, is it necessary to include "Future versions..." at all?

Reply:
The order of plugins matters in some scenarios. I wanted to make it clear that it might change during cluster lifetime (even if it's exactly the same Z in 3.11.Z).

The customer doesn't know when we perform upgrades, so they can't prepare for a scheduled change.